bower.json

```diff
@@ -24,10 +24,10 @@
     "preview.png"
   ],
   "dependencies": {
-    "polymer": "polymer/polymer#^1.1.0"
+    "polymer": "polymer/polymer#1.9 - 2"
   },
   "resolutions": {
-    "polymer": "^1.1.0",
+    "polymer": "~1.9",
     "webcomponentsjs": "^0.7.18"
   }
```

Comment:
Aren't bower's resolutins taken into consideration only in case of conflict? I'm genuinely asking. https://jaketrent.com/post/bower-resolutions/#bower-json-resolutions